### ISSUE TITLE
Something missing in code?

### ISSUE THREAD

**Issue by hseliger** [BenSt099/isosafety#2]
Hi!
I was just trying to use the package but got strange commands about missing $ signs. So checked the code and it seems the `\Isosign` macro only spits out the path and name of the pdf-file, but is missing the graphixs command (graphixs imported via \RequirePackage, so I assume that what was supposed to be used) to actually typeset the image.

Or am I missing something here?

Also, to match the location of the pdf files, the documentation should be updated. It says to use e.g. \Isosign{ISO-F001.png}, but in reality it should be \Isosign{F001}

And lastly, at least on my system (texlive2024 on linux) the PDFs cannot be found in the end, as pdf files seem not to get returned by kpsearch. I assume because it will look in the macro folders (under texmf-dist/tex//) and *not* under texmf-dist/doc//

Any comments? Maybe I'm mistaken…

– Hendrik

**Comment by BenSt099** [comment]
Hello Hendrik,
thanks for reporting! I will investigate this issue as soon as possible and will inform you about that.

- Ben

**Comment by BenSt099** [comment]
I experienced similar issues regarding this package; for a quick fix:
In the folder _temf-dist/tex/latex/isosafety_, open the .sty file and add the full path:
`\newcommand{\Isosign}[1]{/texlive/2024/texmf-dist/doc/latex/isosafety/isosafety-pdfs/ISO_7010_#1.pdf}`. You're right about the paths. The pfds are in _/doc_ (not were they should be). I will try to fix this. Regarding the _\RequirePackage{graphix}_: it is actually unnecessary and will be removed. The syntax in the documentation, however, is correct. The idea is that you can include the pdf like an image an resize it if necessary

**Comment by hseliger** [comment]
Understand. Thanks for the response! So moving the pdf files to the correct folder should sort out them not being found.

I'd suggest to still change the documentation in two ways:

1. Change the syntax from \Isosign{ISO-F001.png} to \Isosign{F001}, as that is what seems to be coded
2. Make clear that the macro returns the full file name, not the image, maybe by adding an example how to typeset it using e.g. graphixs.

Cheers,
Hendrik

**Comment by BenSt099** [comment]
Hey, I fixed / tried to bypass the problem in the newest release. The package and the updated documentation is on its way to CTAN. The actual problem was masked by my system (In the example that I tried on my system, I used the local .sty file in my folder and not the one in my installation), so I didn't notice it in version 1.1. Anyway,  thanks for reporting this major bug and thank you for your feedback! 
If there is any problem again or still, just contact me.
Have a nice day!
Ben
